### PR TITLE
Add screenshot and dialog IPC methods

### PR DESCRIPTION
**Pull request by jonahkagan**
Used by the dev dock (see https://github.com/votingworks/vxsuite/pull/3314)

### CODE REVIEW

File: src/ipc/dialog.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export const channel = 'dialog';
+
+export default function register(ipcMain: IpcMain): void {
+  ipcMain.handle(
+    channel,
+    async (event: IpcMainInvokeEvent, options: OpenDialogOptions) => {
+      return dialog.showOpenDialog(options);
+    },
+  );
+}
```
Comment (adghayes) [inline src/ipc/dialog.ts:17]: When I first wrote this, I was only thinking of the "Open" dialog, whereas I could have built this in such a way that also gives us access to the "Save" dialog. Using the "Save" dialog for saving the screenshot may be cleaner than using the link generated method. 

I think we should either a) change this to enable more dialogs (through arguments or separate endpoints) or b) rename this one to be specific to the open dialog. 

I'm happy to do this.

File: src/ipc/screenshot.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { IpcMain, IpcMainInvokeEvent } from 'electron';
+
+export const channel = 'screenshot';
+
+export default function register(ipcMain: IpcMain): void {
+  ipcMain.handle(channel, async (event: IpcMainInvokeEvent) => {
+    const nativeImage = await event.sender.capturePage();
+    return nativeImage.toPNG();
```
Comment (adghayes) [inline src/ipc/screenshot.ts:8]: PNG seemed like a safe format but we could make other choices here.

File: src/ipc/dialog.ts (hunk 1 above)
Comment (jonahkagan) [reply src/ipc/dialog.ts:17]: I like the idea of enabling more dialogs!

File: src/ipc/screenshot.ts (hunk 2 above)
Comment (jonahkagan) [reply src/ipc/screenshot.ts:8]: Seems fine to start!

**Review by adghayes — APPROVED** [review]

**Pull request merged**